src/protagonist/DLCS.Repository/Messaging/EngineClient.cs

```diff
@@ -26,7 +28,10 @@ public class EngineClient : IEngineClient
     private readonly HttpClient httpClient;
     private readonly ILogger<EngineClient> logger;
     private readonly DlcsSettings dlcsSettings;
-
+    
+    private const string EngineIngestEndpoint = "asset-ingest";
+    private const string LegacyEngineIngestEndpoint = "image-ingest";

```

Comment:
We do not need to handle this. The whole Legacy handling will be removed in the future but don't introduce new code to handle it as it's a path that's never called.